changelog/unreleased/update-passwod-policies.md

```diff
@@ -0,0 +1,7 @@
+Enhancement: Update password policies
+
+The Password policies have been updated.
+The special characters list became constant.
+OCIS-3767
```

Comment:
please exalate and link it

Reply:
I did add a link because it is a link to the private jira